tests/nfv/prepare_env.pm

```diff
@@ -68,13 +68,18 @@ sub run {
         zypper_call('ar -f http://download.suse.de/ibs/SUSE:/SLE-12:/GA/standard/SUSE:SLE-12:GA.repo SLE-12-GA-REPO');
         assert_script_run("bash -x build_base_machine.sh", 300);
     }
+    elsif (check_var('VERSION', '15-SP1')) {
+        assert_script_run("sed -n -e :a -e '1,8!{P;N;D;};N;ba' -i build_base_machine.sh");
+        assert_script_run("cp -r sles/15 sles/15.1");
```

Comment:
if you will have a separate branch for each SP you will kill your self really soon :) 
1. looks like first and last lines are the same with SLE 15 , right ?
2. didn't get why you don't need same repo as you do for SLE 15 ?
3. why you need to copy something from folder with name 15 to folder with name 15.1 ? will test fail in case you build it in sles/15  ? :)

Reply:
1) I reordered things a bit. 
2) Eventually, adding that repo is not needed in 15.1, I am checking on 15.0
3) Cause Vsperf complains that there is no directory named 15.1 taken from the system version.